adapters/handlers/grpc/v1/batch_parse_request_test.go

```diff
@@ -338,6 +341,41 @@ func TestGRPCBatchRequest(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name: "named multi vectors with 1 token level embedding",
```

Comment:
can you add a case of the normal non-colbert vector type?

Reply:
added a bunch of tests that cover all scenarios